.github/workflows/gh-pages.yaml

```diff
@@ -17,6 +17,7 @@ jobs:
         run: |
           yarn --version
           yarn install
+          yarn exec turbo -- run build --filter="./packages/manager/core/*"
```

Comment:
**question:** Should the entire workspace need to be build for generating the documentation?

Reply:
Ok I prepare a PR to fix it 👍